src/components/partials/profile/SendTokenModal.vue

```diff
@@ -145,25 +171,36 @@ export default class SendTokenModal extends Vue {
     async sendToken() {
         this.error = "";
         if (this.validate()) {
-            try {
-                await this.store.dispatch("SEND_TOKEN", {
-                    chain: this.identity.chain?.toLowerCase(),
-                    senderAddress: this.selectedAccount,
-                    receiverAddress: this.identity.basicInfo.address,
-                    //--TODO -- calculate correct amount based on chain decimals
-                    amount: this.tokenAmount
-                });
-            } catch (error) {
-                console.log(error.message);
-            }
+            await this.store.dispatch("SEND_TOKEN", {
```

Comment:
not sure we should remove the try/catch here. The SEND_TOKEN function might still throw an error that we need to catch, e.g. when there is an issue with the wsAddress or the node is not available, ...